src/schema/__tests__/show.test.js

```diff
@@ -767,4 +767,125 @@ describe("Show type", () => {
       })
     })
   })
+
+  describe("#artworks_connection", () => {
```

Comment:
@mzikherman - updated the `show` type with the modern connection form and added a test